README.md

```diff
@@ -213,7 +212,12 @@ The `project_id` for API requests can be created under [blockfrost.io](https://b
 
 **Recommended:** Follow this [approach](https://github.com/lxieyang/chrome-extension-boilerplate-react#secrets) in order to keep the keys seperate from the repository.
 
-The quick solution is to go under `./src/config/provider.js` and replace `secrets.PROJECT_ID_MAINNET` and `secrets.PROJECT_ID_TESTNET` with the project ids from blockfrost.
+```
+# Update secrets file with your own keys
+cp secrets.testing.js secrets.development.js
```

Comment:
please add `secrets.production.js` too here, because otherwise the `npm run build` command still fails